components/brave_ads/core/internal/account/user_data/README.md

```diff
@@ -1,45 +1,54 @@
 
-### Additional User Data Submitted for Brave Rewards Users
+| key  | example  | description  |
+|---|---|---|
+| conversion.action[^2]  | {"conversion":[{"action":"click"}]}  | Click-through conversion attribution to determine which advertising efforts led a user to click on an ad and subsequently complete a desired action.  |
 
-Should only be included in the confirmation payload for users who have joined Brave Rewards.
+## Brave Rewards User
 
-| user data  | optional  | mutable  | description  |
-|---|---|---|---|
-| buildChannel  | no  | no  | Browser build channel.  |
-| catalog  | no  | no  |  Catalog identifier.  |
-| createdAtTimestamp  | no  | no  | Privacy-preserving [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) timestamp.  |
-| diagnosticId  | yes  | yes  | Diagnostic id from [brave://rewards-internals](brave://rewards-internals) to help diagnose issues.  |
-| httpResponseStatus  | yes  | no  | Whether the navigation committed an error page.  |
-| locale  | yes  | no  | Privacy-preserving operating system locale  |
-| platform  | no  | no  | Operating system.  |
-| rotatingHash  | yes  | no  | Privacy-preserving time-based rotating hash.  |
-| segment  | yes  | no  | Advertising taxonomy for the chosen ad.  |
-| studies  | no  | no  | User studies. See [Griffin](https://github.com/brave/brave-browser/wiki/Brave-Variations-(Griffin)).  |
-| systemTimestamp  | no  | yes  | Privacy-preserving [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) timestamp.  |
-| topSegment  | yes  | no  | Top user interest segment. |
-| versionNumber  | no  | no  | Browser version number.  |
+### User Data Submitted when Redeeming Anonymous Confirmation Tokens
 
-## User Data Submitted when Redeeming Identifiable Payment Tokens
+Included as part of the "confirmation token" payload. See [Brave Rewards user confirmation token payload](../confirmations/reward/README.md).
 
-See [identifiable payment token redemption](../utility/redeem_payment_tokens/README.md).
+| key  | example  | description  |
+|---|---|---|
+| buildChannel  | {"buildChannel": "beta"}  | Browser build channel.  |
+| catalog  | {"catalog":[{"id":"29e5c8bc0ba319069980bb390d8e8f9b58c05a20"}]}  |  Catalog identifier.  |
+| conversion.action[^2]  | {"conversion":[{"action":"view"}]}  | View-through or click-through conversion attribution to determine which advertising efforts led a user to view or click on an ad and subsequently complete a desired action.  |
+| conversion.envelope[^2]  | {"conversion":[{"envelope":<br>{"alg":"crypto_box_curve25519xsalsa20poly1305",<br>"ciphertext":"3f8a8aeb5e1e4b1a8e7b4e3f8a8aeb5e1e4b1a8e7b4e3f8a8aeb5e1e4b1a8e7b",<br>"epk":"d1f2e3d4c5b6a79887a6b5c4d3e2f1d1f2e3d4c5b6a79887a6",<br>"nonce":"a1b2c3d4e5f6a7b8c9d0e1f2a3b4c5d6"}}]}  | Allow advertisers to affirm that conversions are legitimate by wrapping an identifier with an additional layer of encryption to protect its integrity.  |
```

Comment:
Have we published how Verifiable Conversions work anywhere? Can we link to that?

Reply:
Added. Thanks